src/analyze_bi.js

```diff
@@ -110,7 +110,7 @@ function processlistEN(cloudlist) {
     for (i = 0; i < cloudlist.length; i++) {
         var validkey = 1
         for (j = 0; j < members.length; j++) {
-            if ((members[j] == cloudlist[i][0]) || members[j].includes(cloudlist[i][0]) || cloudlist[i][0].includes(members[j]) || cloudlist[i][0].includes("Sticker") || cloudlist[i][0].includes("Photo") || cloudlist[i][0].includes("AM") || cloudlist[i][0].includes("PM") || cloudlist[i][0].includes("Call") || cloudlist[i][0].includes("未接來電") || cloudlist[i][0].includes("am")|| cloudlist[i][0].includes("pm")|| cloudlist[i][0].includes("time")) {
+            if ((members[j] == cloudlist[i][0]) || members[j].includes(cloudlist[i][0]) || cloudlist[i][0].includes(members[j]) || cloudlist[i][0].includes("ticker") || cloudlist[i][0].includes("Photo") || cloudlist[i][0].includes("AM") || cloudlist[i][0].includes("PM") || cloudlist[i][0].includes("Call") || cloudlist[i][0].includes("未接來電") || cloudlist[i][0].includes("am")|| cloudlist[i][0].includes("pm")|| cloudlist[i][0].includes("time")) {
```

Comment:
Hi, did you make a typo on the word "Sticker"? Or somehow you did it on purpose?

Reply:
I only put ticker so that it will match both uppercase Sticker and lowercase sticker

Reply:
Oh sure. Any idea why sticker will appear in both upper and lower case, but "Photo" won't?

Reply:
No idea, almost all of them are uppercase, but I did run into a few lowercase ones.
Didn't see any lowercase photo so didn't take the same measure on it.